lib/fork.js

```diff
@@ -2,7 +2,11 @@ const childProcess = require('child_process')
     , childModule  = require.resolve('./child/index')
 
 function fork (forkModule) {
-  var child = childProcess.fork(childModule, {
+  // suppress --debug flags while preserving others (like --harmony)
+  var filteredArgs = process.execArgv.filter(function (v) {
+        return v == '--debug'
+      })
+    , child        = childProcess.fork(childModule, { execArgv: filteredArgs }, {
```

Comment:
I think that either `execArgv` should be in the options list specified as the last argument or you should just supply `filteredArgs` as the second argument (plain, as an array). The former is probably the best call, I think, tho you might want to test!